.release-please-manifest.json

```diff
@@ -1,5 +1,8 @@
 {
   "wnfs": "0.1.16",
-  "wnfs-wasm": "0.1.16",
-  "wnfs-bench": "0.1.16"
+  "wnfs-bench": "0.1.16",
+  "wnfs-common": "0.1.16",
+  "wnfs-hamt": "0.1.16",
+  "wnfs-namefilter": "0.1.16",
```

Comment:
Ah - I wish I said this sooner so I could've spared you some work: We probably don't need to extract out the namefilter, because (1) it's just a small wrapper around what should eventually be a dependency on the [deterministic-bloom](https://github.com/wnfs-wg/deterministic-bloom) package, and (2) we'll remove the namefilter code once we implement name accumulators, which will happen in the next 2-3 weeks (I'm working on the spec ATM).

Reply:
I'm aware of this. It wasn't hard to move out at all. My thinking was that it no longer needs to live in `wnfs` core just like the `deterministic-bloom` package.
So the transition to another dependnency should be easy.